encord/ontology.py

```diff
@@ -12,9 +17,14 @@ class Ontology:
     :meth:`encord.user_client.EncordUserClient.get_ontology()`
     """
 
-    def __init__(self, querier: Querier, instance: OrmOntology):
+    def __init__(
+        self, querier: Querier, instance: OrmOntology, skeleton_templates: Optional[list[SkeletonTemplate]] = None
```

Comment:
As is, the only time we have the skeleton_templates is when we create a project using them. They aren't for instance pulled down on get_ontology. Alexey has expressed an interest against SkeletonTemplates being pulled in a different query but that feels like quite a lot of effort.